### PR TITLE
Update interchain-security.md

### PR DESCRIPTION
**Pull request by okwme**
Closes: #XXX

## Description

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/main/README.md#merging-a-pr))

### CODE REVIEW

File: docs/interchain-security.md
Comment (jtremback) [inline docs/interchain-security.md:54]: > This could be purely the chance to receive fees on a new network, it could be paired with some genesis distribution of tokens

It's true that all of these things are viable and will be tried, but I think for the sake of clarity in this paper, I would emphasize the basic mechanism that we are likely to build into the protocol: the ability for consumer chains to offer a percentage of their staking rewards to the validators/delegators staked on the provider chain. This will keep the message focused and understandable.

> it could be paired with some off chain legal agreement to pay for the staking services

I removed this because while it is indeed possible, and will probably happen, I don't know why we emphasize it. Makes things sounds kinda centralized.

```suggestion
Once V1 has been finalized the next milestone would be to create a version of the Chain Registry where Provider Chain validators opt-in to participating as a validator on the Consumer Chain. It is expected that the candidates for Opt-In Interchain Security are not considered part of the Cosmos Hub itself, but independent networks. It is up to each Consumer Chain to advertise the benefits that Provider Chain validators receive for their participation. Consumer chains will be able to offer a certain percentage of their fees and staking rewards to provider chain validators (and their delegators) that validate on them. Chains will also be able to launch as consumer chains, in which case they could offer starting validators and delegators on the provider chain a percentage of their genesis tokens. 
```

File: docs/interchain-security.md
Comment (jtremback) [inline docs/interchain-security.md:58]: I kind of want the "base case" of interchain security to be that an already running chain starts consuming security from the provider chain.

I agree that having interchain security as part of a chain's launch will be a very important use case but it should be a special case (after v1).

```suggestion
The Chain Registry will also include tooling to allow chains to launch as consumer chains. Launching as a consumer chain is great, because the new blockchain gets full security from day one, and is able to get its tokens into the hands of a lot of users. When launching as a consumer chain, projects will set a Time Til Launch (TTL) for their prospective chain in the Chain Registry. The TTL designates the point of no return for Validators to join or leave the Chain Registry. At that point the list of validators contained within the registry for a specific network can be exported to an initial genesis file for that network.
```

File: docs/interchain-security.md
Comment (jtremback) [inline docs/interchain-security.md:60]:
```suggestion
```
See comment on the line above

File: docs/interchain-security.md
Comment (jtremback) [inline docs/interchain-security.md:19]: My preference would be not to uppercase Consumer Chain and Provider Chain. It might be appropriate in a more technical spec, but in this explanatory document I think it gives the wrong impression, like being a Consumer Chain or a Provider Chain is binary, final choice. In fact, consuming or providing security will be an option for any chain at any time, simply by passing a governance proposal.

File: docs/interchain-security.md
Comment (jtremback) [inline docs/interchain-security.md:43]:
```suggestion
How Interchain Security works at a technical level is still in the process of development but the stack at a high level is well mapped out. It requires new functionality and modifications to current functionality in the Cosmos SDK. The technology can be developed progressively so that a minimum viable set of functionality can be launched as a V1 before an extended set of functionality is launched at a later date.
```

File: docs/interchain-security.md
Comment (jtremback) [inline docs/interchain-security.md:69]:
```suggestion
```

Too technical for this paper, IMO, and this also isn't how we're doing it AFAIK

File: docs/interchain-security.md
Comment (jtremback) [inline docs/interchain-security.md:67]:
```suggestion
```

Too technical for this paper, IMO, and this also isn't how we're doing it AFAIK

**Review by jackzampolin — APPROVED** [review]
Reading through in some detail and learning quite a bit. LGTM

File: docs/interchain-security.md
Comment (jtremback) [inline docs/interchain-security.md:85]:
```suggestion
```

> Initially the cost to validators to operate a new node instance on a new network may prohibit smaller validators from participating.

So many small validators run all kinds of stuff. Big validators will not be able to just run everything that comes along because the software poses a risk to their systems. An ecosystem naturally will develop where small validators take bigger risks starting out to get established.

This whole Chain-Specific Delegations section needs to be taken out of this paper for now IMO.  It's a second-generation feature. It's very hard to understand for someone who is not familiar with interchain security (or steeped in the Cosmos core dev bubble) and it is not on our near-term roadmap.

File: docs/interchain-security.md
Comment (jtremback) [inline docs/interchain-security.md:117]: > The xStaking module

inaccurate and not especially interesting from a high level perspective

File: docs/interchain-security.md
Comment (jtremback) [inline docs/interchain-security.md:141]: I agree with this section but I think it is way too long. In short, here's how I think things will look: 

Downtime slashing will be initiated by the consumer chain, while double signing slashing is reported directly to the provider chain. This is because double signing is trivially verifiable, while downtime is not verifiable at all. 

Consider: If malicious validators on a consumer chain wanted to falsely slash another validator for downtime, they could simply censor all of the target validator's messages with 1/3 of the power, forcing real downtime. Meanwhile, sending some kind of fake downtime slash request packet to the provider would take 2/3 of the power.

**Review by jtremback — CHANGES_REQUESTED** [review]
I have a lot more changes I want to make but the Github PR UI isn't the best place to make them. I will make a new PR after this one is merged, with my additional suggested changes

**Pull request merged**
